homeassistant/components/airly/__init__.py

```diff
@@ -53,6 +54,7 @@ async def async_setup_entry(hass, config_entry):
     api_key = config_entry.data[CONF_API_KEY]
     latitude = config_entry.data[CONF_LATITUDE]
     longitude = config_entry.data[CONF_LONGITUDE]
+    use_nearest = config_entry.data.get(CONF_USE_NEAREST, False)
```

Comment:
This is the right approach. We should try to avoid config entry format changes as much as possible as it is difficult to roll back.